src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -1415,7 +1415,15 @@ function setupFsCopyModal() {
                 for (var i = 0 ; i < coursesTable.length; i++) {
                     htmlToAppend += "<div class=\"checkbox\">";
                     htmlToAppend += "<label><input type=\"checkbox\" name=\"copiedcoursesid\"";
-                    htmlToAppend += "value=\"" + coursesTable[i].id + "\"> [" + coursesTable[i].id + "] : " + coursesTable[i].name;
+                    if (String(coursesTable[i].id) == courseid){
+                    	htmlToAppend += "value=\"" + coursesTable[i].id + "\"> [" + "<font color=\"red\">" + coursesTable[i].id + "</font>" + "] : " + coursesTable[i].name;
+                    }
```

Comment:
Similar to what I commented on in the other issue, "else" should be written on the same line as the closing curly brackets. Sorry for not spotting it earlier.